react-graphql-apollo-tutorial.md

```diff
@@ -371,45 +371,45 @@ It's up to you to come up with an own folder/file structure for your components
+In this section, you will set up a Apollo Client instance in the Node.js environment, same as before. However, this time you will use Apollo Client directly without the zero-configuration package Apollo Boost, meaning you'll need to configure the Apollo Client yourself without sensible defaults. While it's best to use a tool with sensible defaults for learning, configuring Apollo yourself exposes the composable ecosystem of Apollo Client, how to use it for an initial setup, and how to advance this setup later. 
 
-All the Apollo Client setup can be done in the top level *src/index.js* file of your project where you have your React to HTML entry point as well. Before you touch this file, you have to install the Apollo Client on the command line in your project folder:
+The Apollo Client setup can be completed in the top-level *src/index.js* file, where the React to HTML entry point exists as well. First, install the Apollo Client in your project folder using the command line:
 
 {{< highlight javascript >}}
 npm install apollo-client --save
 {{< /highlight >}}
 
-Furthermore, you need two utility packages for two mandatory configurations that are used for the Apollo Client creation. Whereas the {{% a_blank "apollo-cache-inmemory" "https://github.com/apollographql/apollo-client/tree/master/packages/apollo-cache-inmemory" %}} is a recommended cache (read also as: store or state) for your Apollo Client to manage the data, apollo-link-http is used to configure the URI and additional network information once for your Apollo Client instance.
+Two utility packages are requied for two mandatory configurations used to create the Apollo Client. The {{% a_blank "apollo-cache-inmemory" "https://github.com/apollographql/apollo-client/tree/master/packages/apollo-cache-inmemory" %}} is a recommended cache (read also as: store or state) for your Apollo Client to manage the data, while apollo-link-http is used to configure the URI and additional network information once for an Apollo Client instance.
 
 {{< highlight javascript >}}
 npm install apollo-cache-inmemory apollo-link-http --save
 {{< /highlight >}}
 
-As you can see, their is nothing yet mentioned about React. It's only the Apollo Client plus two packages for the configuration of it. Moreover, there are two more packages that you have to install for Apollo Client to work with GraphQL. They are used as internal dependencies by Apollo. However, the latter one is also used to define your queries and mutations later on. Before these utilities came directly from Apollo Boost as you may recall.
+As you can see, nothing has been mentioned about React, only the Apollo Client plus two packages for its configuration. There are two additional packages required for Apollo Client to work with GraphQL, to be used as internal dependencies by Apollo. The latter is also used to define queries and mutations. These utilities came directly from Apollo Boost.
```

Comment:
"These utilities came directly from Apollo Boost." to "Previously, these utilities came directly from Apollo Boost."